aquadoggo/src/rpc/methods/publish_entry.rs

```diff
@@ -20,8 +20,11 @@ pub enum PublishEntryError {
     #[error("Could not find skiplink entry in database")]
     SkiplinkMissing,
 
-    #[error("Claimed log_id for schema not the same as in database")]
-    InvalidLogId,
+    #[error("Requested log id {0} does not match expected log id {1}")]
+    InvalidLogId(String, String),
```

Comment:
Do these values need to be a `String`? `i64` should also implement `Display`.

Reply:
Good point! I will change that. I just didn't want to use the display format of `LogId`, which would have been something like `LogId(3)`.